### PR TITLE
[YVR-Common-33] Connect a Backend to Client

### PR DESCRIPTION
**Pull request by jasper-oh**
https://trello.com/c/6YTN1LcL

Jeen's branch yvr-common-33 can be deleted

### CODE REVIEW

File: components/Clickable.tsx
Comment (jasper-oh) [inline components/Clickable.tsx:1]: I got error in line 21 in this components. Can someone look at it? @gdcho @JunyoungLee0421 @EunjeongHur @JeenNamkung @ys421

Comment (gdcho) [reply components/Clickable.tsx:1]: That is just typescript type error

**Review by gdcho — APPROVED** [review]
lgtm

File: components/Clickable.tsx
Comment (jasper-oh) [reply components/Clickable.tsx:1]: Oh we can ignore the type error? Or is this related to configuration caches

Comment (gdcho) [reply components/Clickable.tsx:1]: I'm pretty sure we can ignore but I can look into it again

Comment (gdcho) [reply components/Clickable.tsx:1]: Looks like @ys421 fixed it in his branch

**Pull request merged**